src/hooks/v3/useAllV3Routes.ts

```diff
@@ -0,0 +1,107 @@
+import { Currency } from '@uniswap/sdk-core';
+import { Pool, Route } from 'v3lib/entities';
```

Comment:
We need to condensed this o have this whole folder duplicated on my branch it will cause a lot of conflicts let's coordinate the move

Reply:
Yes, let's go with v3Lib,  for now.  After that in one commit let's remove all duplicates.

Reply:
Also, I'm going to close this pr and create a new pr, 
I'll create a new branch from your latest V3-Implementation branch and cherry-pick my new changes into it,